### PR TITLE
added utf8mb4_support

### PR DESCRIPTION
**Pull request by Areezy**
Added support for utf8mb4 character set in the users and readings tables while keeping the byte size the same for the columns.

### CODE REVIEW

**Review by aubundy — CHANGES_REQUESTED** [review]
Local env db tables and seed data is set up through the current `import.sql` and `tables.sql` files. With these alter statements being in a separate file, these changes won't make it to any local dev environment. Can you add these alter statements to the bottom of `tables.sql`? Please then run 

```sql
-- create tables and import seed data
mysql -u [USERNAME] -p [DATABASE] < mysql/tables.sql 
mysql -u [USERNAME] -p [DATABASE] < mysql/import.sql
```

to confirm the changes are set up properly

**Comment by Areezy** [comment]
Okay, I will do that and then update the pull request.

**Comment by Areezy** [comment]
I added the alter statements to the tables file and also tested it and confirmed it works.

File: mysql/tables.sql [hunk 1]
```diff
@@ -99,4 +99,36 @@ CREATE TABLE user_tags (
...
   CONSTRAINT user_tags_ibfk_1 FOREIGN KEY (user_id) REFERENCES users (id),
   CONSTRAINT user_tags_ibfk_2 FOREIGN KEY (tag_id) REFERENCES tags (id)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8;
\ No newline at end of file
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+-- Alter database for new tables
+ALTER DATABASE
+    bread_server
```
Comment (aubundy) [inline mysql/tables.sql]: @Areezy Each user will name their local db something unique, can you replace this to use the environment variable for db? `process.env.LOCAL_DB`

**Review by aubundy — CHANGES_REQUESTED** [review]

File: mysql/tables.sql (hunk 1 above)
Comment (Areezy) [reply mysql/tables.sql]: That has been done now. Thanks for the heads-up!

**Pull request merged**